### DISCUSSION TITLE
On campus jobs

### DISCUSSION THREAD

**Discussion by palash-98** [snigi-gupta/AttendingUniversityAtBuffalo#58] [Q&A] [answered]
What is the process of applying for jobs on campus other than handshake? Also how can you approach undergrad department for grading and other jobs like TA?

**Comment by snigi-gupta** [comment, marked as answer]
@palash-98 
The best way is Handshake tbh. However, if you network well and get in touch with people who are already doing on campus jobs, there is a high chance of you getting a referral and getting selected. 
For TA jobs, best way is to talk to the current TA and see if the prof needs more TAs, talk to the prof, do great in their class (profs love that), and just try to show that you can make their life easier.

For undergrad, I am not really sure, but you can check out the UB websites, to find a POC for this task. 

I hope this helps. Sorry for the delay in replying (I was on a tour).
Please follow me at https://www.instagram.com/yourtechdiary/ to learn hacks to secure a job and more!
